themes/qgrid/source/js/menu.js

```diff
@@ -32,21 +32,35 @@ function filter(e) {
 	}
 }
 
+function showSearch(e) {
+	var target = e.target || e.srcElement;
```

Comment:
here `target` doesn't say what should be hidden, is it trigger button? can we just assing appropriate class like '.active' and hide button by css? and show search by css too